jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/openshift/RouteEnricher.java

```diff
@@ -72,6 +76,11 @@ private boolean isGenerateRoute() {
         return Configs.asBoolean(getConfigWithFallback(Config.GENERATE_ROUTE, GENERATE_ROUTE_PROPERTY, null));
     }
 
+    private boolean isRouteWithTLS(){
+        String propval = getConfigWithFallback(Config.TLS_TERMINATION, GENERATE_TLS_TERMINATION_PROPERTY, null);
+        return (propval != null && !propval.isEmpty());
```

Comment:
Please Use `StringUtils#isNotBlank`

Reply:
Sorry, didn't see these comments. Will fix and rebase